tests/test-smith-form-local.C

```diff
@@ -155,28 +153,31 @@ static bool testLocalSmith (const LocalPIR &R, vector<typename LocalPIR::Element
 	return ret;
 }
 
-int main (int argc, char **argv)
-{
-	bool pass1 = true, pass2 = true;
 
-	static int64_t n = 6;
-	static size_t q = 3; 
-	static int32_t e = 4;
-	//static integer q = 10201; // 101^2
+int main (int argc, char **argv) {
+    bool pass1(true), pass2(true);
+    static int64_t n = 6;
+    static int  q = 3;
+    static int32_t e = 4;
+    static int rseed = (int)time(NULL);
 
 	static Argument args[] = {
 		{ 'n', "-n N", "Set dimension of test matrices to NxN.", TYPE_INT,     &n },
 		{ 'q', "-q Q", "Operate over the ring Z/q^eZ.", TYPE_INT, &q },
-		//{ 'q', "-q Q", "Operate over the ring Z/qZ.", TYPE_INTEGER, &q },
+        { 'e', "-e e", "Operate over the ring Z/q^eZ.", TYPE_INT, &e },
+        { 's', "-s S", "Random generator seed.", TYPE_INT,     &rseed }	,
 		END_OF_ARGUMENTS
 	};
 
 	parseArguments (argc, argv, args);
+    std::srand(rseed);
+    FFLAS::writeCommandString(std::cout, args) << std::endl;
```

Comment:
This breaks the standard output interface of LinBox test suite (and will cause the CI to fail)